var/spack/repos/builtin/packages/py-fava/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFava(PythonPackage):
+    """Fava is a web interface for the double-entry bookkeeping software
+       Beancount with a focus on features and usability."""
+
+    homepage = "https://beancount.github.io/fava/"
+    pypi     = "fava/fava-1.18.tar.gz"
+
+    version('1.18',  sha256='21336b695708497e6f00cab77135b174c51feb2713b657e0e208282960885bf5')
+
+    # For some reason Fava adds a whole bunch of executables to
+    # its bin directory, and this causes clashes when loading
+    # the module.
+    extends('python', ignore='bin/(bean-*|chardetect|cheroot|flask|py*|treeify|upload-to-sheets)')
+
+    # Some of the dependencies are not listed as required at
+    # build or run time, but actually are.
+    # - py-setuptools
+    # - py-importlib
+    # - py-pytest
+    depends_on('python@3.6:',       type=('build', 'run'))
+    depends_on('py-setuptools',     type='build')
```

Comment:
I think you'll want this at run-time too:
```
$ grep -R pkg_resources *
src/fava/__init__.py:from pkg_resources import DistributionNotFound
src/fava/__init__.py:from pkg_resources import get_distribution
```

Reply:
Thank you!